package.json

```diff
@@ -19,7 +19,10 @@
     "lint": "gulp lint",
     "build": "gulp build",
     "dist": "gulp dist",
-    "heroku-postbuild": "gulp clean && gulp build && gulp dist --fortesting"
+    "heroku-postbuild": "gulp clean && gulp build --fortesting && gulp dist --fortesting && npm run try-prepend",
+    "try-prepend": "if [ -n \"$AMP_TESTING_HOST\" ]; then npm run prepend-amp; npm run prepend-v0; fi",
+    "prepend-amp": "gulp prepend-global --prod node_modules/AMP_CONFIG.json --local --target dist/amp.js && gulp prepend-global --prod node_modules/AMP_CONFIG.json --local --target dist.3p/current/integration.js",
+    "prepend-v0": "gulp prepend-global --prod node_modules/AMP_CONFIG.json --local --target dist/v0.js && gulp prepend-global --prod node_modules/AMP_CONFIG.json --local --target dist.3p/current-min/f.js"
```

Comment:
sorry, meant to clarify. on cdn.ampproject.org we don't need to do the prepend f.js since the values we want are the default but for any other cache they would need to prepend the AMP_CONFIG to f.js, so just an update to the SERVING.md file is required